ActindoWorkFlow/UserInterface/05a_QueueTypes.md

```diff
@@ -42,20 +44,21 @@ The list displays all queue types. All fields are read-only. Depending on the se
     - **Oldest executable action**
     - **Executable actions of oldest process**
 
+- *Throttled*
+     - ![Check](../../Assets/Icons/Check.png "[Check]") (Checkmark): The queue type is defined to throttle a process.
+    - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross): The queue type is not defined to throttle a process. 
+
 - ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
     Click this button to create a queue type. The *Create queue type* view is displayed, see [Create a queue type](../Integration/01_ConfigureQueueTypes.md#create-a-queue-type).
 
 
 
-## Create queue type
+## Create a queue type
```

Comment:
Fenster- /Ansichtsnamen haben i.d.R. keine Artikel in unserer Doku (und im UI). Vgl. Edit weiter unten, oder "Add comment" in Process actions.